sparkbox/apprentices-page-next
# feat: Change placeholder logo color

## Description
This PR changes the color of the logo in the placeholder image.

## To Test

1. Make sure all PR Checks have passed (Github Actions, Netlify etc).
2. Visit the PR site.

- [ ] Confirm the placeholder image has a black Sparkbox logo.
